recipes/elastic-blast/meta.yaml

```diff
@@ -9,8 +9,8 @@ source:
   sha256: 2e7fe4c312e43b2990fbe3420fd8a67bf5566ea5ace6d400d3e68058038dfa81
 
 build:
-  number: 0 
-  noarch: python
+  number: 0
```

Comment:
Shouldn't build number be updated?

Reply:
That's what I initially thought, but I got build errors when I did that.